.travis.yml

```diff
@@ -0,0 +1,43 @@
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode9.0
+      language: node_js
+      node_js: "6.11.1"
+      env:
+        - ELECTRON_CACHE=$HOME/.cache/electron
+        - ELECTRON_BUILDER_CACHE=$HOME/.cache/electron-builder
+
+cache:
+  directories:
+  - node_modules
+  - $HOME/.cache/electron
+  - $HOME/.cache/electron-builder
+  yarn: true
+
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 0.27.5
+  - export PATH=$HOME/.yarn/bin:$PATH
+
+install:
+  - yarn install
+  - yarn run rebuild:modules:electron
+
+script:
+  - yarn run build
+  - yarn run package
```

Comment:
Just want to make sure that this branch's  `package.js` contains the exception for mac: https://github.com/plotly/falcon-sql-client/blob/master/package.js#L67

Reply:
This is the relevant issue in case you want to go into details: https://github.com/ibmdb/node-ibm_db/issues/329